docs/get-started/Gsoc.md

```diff
@@ -0,0 +1,96 @@
+---
+id: GSOC
+title: GSOC
+---
+
+
+
+# 2022 GSoC Towhee Proposal
+
+### About Towhee
+
+![Towhee](https://towhee.io/assets/img/logo-title.png)
+
+
+
+Towhee is a flexible, application-oriented framework for running machine learning (ML) pipelines. It is a Python project that aims to make greatly simple anything2vec, allowing everyone - from beginner developers to large organizations - to deploy complex pipelines with just a few lines of code.
```

Comment:
'aims to make greatly simple anything2vec' keeps popping up in all our documents @fzliu. Where is everyone pulling it from? Is it on our main readme? Replace it with 'aims to simplify the creation and deployment of complex multi-stage ML tasks'